styleguide/derek/pattern-components/platforms/_platforms.scss

```diff
@@ -16,15 +16,6 @@
   padding-left: 25px;
 }
 
-.rsHomepage-platformH1 {
-  font-size: 4.7rem;
-  font-weight: 700;
-  letter-spacing: .5px;
-  line-height: 4rem;
-  margin: 0;
-  text-transform: uppercase;
-}
-
```

Comment:
Shouldn't we be killing off all the font sizes/line heights/weights from the `.rsHomepage-*` classes? I can't imagine that's on brand anymore.

Reply:
@WidgetsBurritos ```.rsHomepage-``` classes aren't getting used anywhere but I removed the rules anyway. ```cxRiver``` sections were not in the list to update. We can wait to see what Mike  wants there.

Reply:
@runrog,
The `rsHomepage` classes are in use in several places on the managed hosting page: https://www.rackspace.com/managed-hosting

We can ignore the cxRiver stuff for now, but the task was kind of open-ended so it's really to cover anything off-brand on any of the pages he referenced.